### ISSUE TITLE
Site should have a coherent link styling hierarchy (when we use orange v. when we use blue) (user story #307)

### ISSUE THREAD

**Issue by ghing** [denverfoundation/storybase#514]
User Story #307

**Comment by patternleaf** [comment]
in #449 i wrote:

> regarding semantic value, i could envision having the chrome/megamenu links, buttons, and widgets generally all be orange (referring to the orange in the logo), while any links in copy/content are blue. so the rule would be floodlight chrome is orange, floodlight content is blue. that would include story titles. for reduced retina damage, we might make buttons/widgets gray + orange on hover, or something.
> 
> also regarding semantics, i could see cases where we might want to call out a button or content by switching it around—a big blue button, say—_if_ there's only one instance of such a thing on the page.

and @jwirfs-brock wrote:

> I like the simplicity of @patternleaf's suggestion about having application-type links be orange and content-type links be blue.

@PitonFoundation/developers do we have any new insight/ideas/directions on this?

**Comment by bpawlak** [comment]
I believe that's the direction we settled on, yes.  We'll have to be careful about making the default buttons grey. Since they're fairly flat in design, the grey color (unless it's a darker grey like #444 ) can get lost on the page.  Plus, if it's too light grey, it looks disabled.

Reserving blue for primary actions, however is a great idea...  In the example below, I've darkened the Share button

![image](https://f.cloud.github.com/assets/2250519/153675/b62754ec-75ff-11e2-9a1b-b5625f1fa676.png)
